srcpkgs/pnpm/template

```diff
@@ -0,0 +1,35 @@
+# Template file for 'pnpm'
+pkgname=pnpm
+version=5.8.0
+revision=1
+# We won't use wrksrc, but it's needed to silence "cannot access wrksrc directory" error
+create_wrksrc=yes
+build_style=meta
+hostmakedepends="nodejs jq"
+depends="nodejs>=10.14"
```

Comment:
This version constraint isn't necessary.

Reply:
Removed